upup/pkg/fi/cloudup/defaults.go

```diff
@@ -227,3 +238,36 @@ func incrementIP(ip net.IP, cidr string) (string, error) {
 	}
 	return ip.String(), nil
 }
+
+// setupEctdVersion enables the supported Etcd version, if not set.
+func setupEctdVersion(cluster *kops.Cluster) error {
+	parsedVersion, err := util.ParseKubernetesVersion(cluster.Spec.KubernetesVersion)
+	if err != nil {
+		return fmt.Errorf("error parsing version: %v", cluster.Spec.KubernetesVersion)
+	}
+
+	version := ""
+	// as more versions are release we can add more if else statements here to set version.
+	if util.IsKubernetesGTE("1.9.0", *parsedVersion) {
+		// https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md#external-dependencies
+		// lists 3.1.10 as the supported version for 1.9.x
+		version = "3.1.10"
```

Comment:
Could we default to the v3.3.x release as this includes quite an important fix for the `mvcc: database space exceeded` error: https://github.com/coreos/etcd/blob/master/CHANGELOG-3.3.md#v330-2018-02-01